api/libproject/devstart.rb

```diff
@@ -488,10 +487,22 @@ def connect_to_db()
 
 Common.register_command({
   :invocation => "connect-to-db",
-  :description => "Connect to the running database via mysql.",
+  :description => "Connect to the running RW database via mysql.",
   :fn => ->() { connect_to_db() }
 })
 
+def connect_to_cdr_db()
+  common = Common.new
+
+  cmd = "MYSQL_PWD=root-notasecret mysql --database=cdr"
+  common.run_inline %W{docker-compose exec db sh -c #{cmd}}
+end
+
+Common.register_command({
+  :invocation => "connect-to-cdr-db",
+  :description => "Connect to the running CDR database via mysql.",
+  :fn => ->() { connect_to_cdr_db() }
```

Comment:
I think a flag would be better to avoid command proliferation. Or just use the existing command:

```
./project connect-to-db
USE cdr;
```

Reply:
ok added a param